### ISSUE TITLE
Error in KeychainAccess for Xcode Version 8.0 beta 6 

### ISSUE THREAD

**Issue by shrawan2015** [kishikawakatsumi/KeychainAccess#260]
Xcode :Version 8.0 beta 6 
Cocopods:/cocoapods-1.0.1
KeychainAccess version : KeychainAccess 2.3.6
Swift : 3.0

![screen shot 2016-09-02 at 8 30 06 pm](https://cloud.githubusercontent.com/assets/10405305/18208206/1cb02b62-714c-11e6-9c70-b21dd8f1c344.png)

Is there any issue in converting the Existing code to swift-3. There are lot's of error in current version of KeychainAccess after compilation .

Could you help for above scenario .

**Comment by kishikawakatsumi** [comment]
@shrawan2015 Use `swift-3.0` branch to use this lib in Swift 3 and Xcode beta.

**Comment by shrawan2015** [comment]
Thanks . Still there is error in  KeychainAccess for swift3 branch and  Xcode 8 GM seed

**Comment by kishikawakatsumi** [comment]
@shrawan2015 You need writing install hook in your Podfile because CocoaPods doesn't respect project build settings.

```
platform :ios, '8.0'

target '...' do
  use_frameworks!
  
  pod 'Realm', git: 'https://github.com/realm/realm-cocoa.git', branch: 'master', submodules: true
  pod 'RealmSwift', git: 'https://github.com/realm/realm-cocoa.git', branch: 'master', submodules: true
  
  post_install do |installer|
    installer.pods_project.targets.each do |target|
      target.build_configurations.each do |config|
        config.build_settings['SWIFT_VERSION'] = '2.3'
      end
    end
  end
  
end
```

But I've released `v3.0.0`, now you can use just `pod 'KeychainAccess'`.
